### DISCUSSION TITLE
Can we hardcode country code in Chat widget?

### DISCUSSION THREAD

**Discussion by manojdhimann** [chatwoot/chatwoot#6649] [General]
**Hi Team,
Can we hardcode the country code in the Chat widget?**

![image](https://user-images.githubusercontent.com/32815842/224256877-66f12a5d-8519-4b91-8c1d-30f267946b1b.png)

**Comment by vishnu-narayanan** [comment]
@manojdhimann No, we don't have support for that yet.

cc: @muhsin-k

**Comment by IlPicasso** [comment]
Or allow us to insert numbers without country code format
